crates/parser/src/zombienet.pest

```diff
@@ -27,6 +27,7 @@ span_id = { ASCII_HEX_DIGIT{32} }
 block_height = { "blockheight" | "block height" | "best block" }
 finalized_height = { "finalised height" | "finalised block" }
 peers_count = { "peers count" | "peers" }
+chaos_type = { "latency" | "jitter" | "correlation" }
```

Comment:
`correlation` manage how `latency/jitter` works internally, I think most of the cases will be `100` (the default value). To not complicate more the D.S.L I would say that we should only allow to set `latency` and `jitter`.